### PR TITLE
add 256 byte vectors to struct elem

### PR DESCRIPTION
**Pull request by WalterBright**
straightforward

### CODE REVIEW

File: src/backend/evalu8.c [hunk 1]
```diff
@@ -270,6 +270,41 @@ int boolres(elem *e)
...
+                case TYuchar32:
+                case TYshort16:
+                case TYushort16:
+                case TYlong8:
+                case TYulong8:
+                case TYllong4:
+                case TYullong4:
+                    b = 0;
+                    for (size_t i = 0; i < 8; i++)
+                        b |= e->EV.Vulong8[i] != 0;
```
Comment (ZombineDev) [inline src/backend/evalu8.c]: Why not operate directly on `Vullong4`, like on line 2211 below?

**Review by PetarKirov — APPROVED** [review]
A part from the question below, LGTM.

**Comment by ZombineDev** [comment]
Just FYI: https://en.m.wikipedia.org/wiki/AVX-512

File: src/backend/evalu8.c (hunk 1 above)
Comment (ZombineDev) [inline src/backend/evalu8.c]: Wouldn't 4 64-bit movs be faster than 8 32-bit ones, at least on x86_64?

Comment (WalterBright) [inline src/backend/evalu8.c]: 1. It's not doing movs.
2. There's a 32 bit dmd build, too.
3. It doesn't matter. This piece of code is nowhere near a hot spot.

**Comment by WalterBright** [comment]
Yes, I know about AVX-512. But I'm not dealing with that at the moment.

**Pull request merged**